src/mots/config.py

```diff
@@ -200,8 +201,25 @@ def clean(file_config: FileConfig, write: bool = True):
 
     bmo_data = get_bmo_data(people)
     updated_people = People(people, bmo_data)
-    logger.debug("Updating people configuration based on BMO data...")
-    file_config.config["people"] = updated_people.serialized
+
+    for person in people:
+        if "nick" not in person:
+            person["sync"] = True
```

Comment:
Instead of adding a flag to the object and the removing it later, we should create a set of IDs where `nick not in person` here and then make the `if` check below a `if person["bmo_id"] in nick_not_in_person_set` etc.

Reply:
This makes sense, will do.